### DISCUSSION TITLE
How to edit the wording of the goals?

### DISCUSSION THREAD

**Discussion by Mazup** [open-sdg/open-sdg#2132] [Q&A] [answered]
Hi,

I think I need help to. I work since some days with open-sdg, and some points are not clear yet. In example the editing the wording of preinstalled goals. The indicators could I edit successful, but I don't no know, how to edit the goal, i.e. 2.1, which not beeing an indicator. This is my data-repository: https://github.com/Mazup/osdg-data

In my repository I cannot find a list or the names or translations of the goals I can see in my frontend: https://mazup.github.io/osdg-site/.

Thanks a lot,
Matthias

**Comment by christoph-LSN** [comment, marked as answer]
@Mazup 
The translations (e.g. indicator names, goal names, etc) come from the translations repo: https://github.com/open-sdg/sdg-translations
You can clone the repo and change this lines https://github.com/Mazup/osdg-data/blob/develop/config_data.yml#L72-L80 your config_data.yml. I have done this in my migration  report too as you can see here: https://github.com/christoph-LSN/IM_Data/blob/develop/config_data.yml#L69

my data repo: https://github.com/christoph-LSN/IM_Data/
my site repo: https://github.com/christoph-LSN/MT_Site
my translations repo: https://github.com/christoph-LSN/IM-translations
my site: https://christoph-lsn.github.io/MT_Site/

**Comment by Mazup** [comment]
Hi Christoph,

thank You very much for nudging in the right direction. It was very helpful to me. 👍 

Matthias
